lib/slim.js

```diff
@@ -1,59 +1,30 @@
 const isWsl = require('is-wsl');
+const glob = require('glob-all');
+const fse = require('fs-extra');
 
-/**
- * Get commands to slim the installed requirements
- * only for non-windows platforms:
- * works for docker builds and when run on UNIX platforms (wsl included)
- * @param {Object} options
- * @param {string} folderPath
- * @return {Array.<String>}
- */
-function getSlimPackageCommands(options, folderPath) {
-  let stripCmd = [];
+const getStripCommand = (options, folderPath) =>
+  process.platform !== 'win32' || isWsl || options.dockerizePip
+    ? ` && find ${folderPath} -name "*.so" -exec strip {} \\;`
```

Comment:
I have noticed that again changing 
`` ` && find ${folderPath} -name "*.so" -exec strip {} \\; ` `` to 
either  `` ` && find ${folderPath} -name "*.so" -exec strip {} \;` `` 
or `` ` && find ${folderPath} -name "*.so" -exec strip {} ;` `` 
makes it run as expected on Win10.

Reply:
Interesting. pushing a commit with out the `\\` to see how it fares in circle. It might have to be a platform dependent thing. I'd assume WSL should be treated like a *nix tho because it's running in as *nix shell.

Reply:
@dschep 
Indeed, I was rather surprised too.. 
If it fails, could be interesting to also replace `;` operator with `+`.  

I did not have a good reason to use a semicolon, but it seems that `+` should also work and it does not generally need an escape: [as mentioned here](https://unix.stackexchange.com/questions/12902/how-to-run-find-exec)